0.19/Dockerfile

```
@@ -1,20 +1,20 @@
-FROM debian:stable-slim
+ARG ARCH=amd64
+FROM debian:stable-slim as build
 
 LABEL maintainer.0="João Fonseca (@joaopaulofonseca)" \
   maintainer.1="Pedro Branco (@pedrobranco)" \
   maintainer.2="Rui Marinho (@ruimarinho)"
 
 RUN useradd -r bitcoin \
   && apt-get update -y \
-  && apt-get install -y curl gnupg gosu \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+  && apt-get install -y curl gnupg
 
+ARG ARCH
 ENV BITCOIN_VERSION=0.19.0.1
-ENV BITCOIN_DATA=/home/bitcoin/.bitcoin
-ENV PATH=/opt/bitcoin-${BITCOIN_VERSION}/bin:$PATH
 
 RUN set -ex \
+  && if [ "${ARCH}" = "amd64" ]; then export ARCH=x86_64; fi \
+  && if [ "${ARCH}" = "arm64v8" ]; then export ARCH=aarch64; fi \
```

Add also arm/v7 architectures. To do that it needs also the different OSTYPE

```suggestion
  && if [ "${ARCH}" = "arm64v8" ]; then export ARCH=aarch64; export OS=linux-gnueabihf; fi \
```